ntbs-integration-tests/NotificationPages/SocialContextAddressEditPageTests.cs

```diff
@@ -224,21 +224,29 @@ public async Task PostDelete_ReturnsSuccessAndRemovesResult()
         public async Task ValidateSocialContextDates_ReturnsErrorIfDateToBeforeDateFrom()
         {
             // Arrange
-            var keyValuePairs = new string[]
+            var initialPage = await Client.GetAsync(GetCurrentPathForId(Utilities.NOTIFICATION_WITH_ADDRESSES) + ADDRESS_ID);
+            var initialDocument = await GetDocumentAsync(initialPage);
+            var keyValuePairs = new DatesValidationModel
             {
-                "keyValuePairs[0][key]=DateFrom",
-                "keyValuePairs[0][day]=1",
-                "keyValuePairs[0][month]=1",
-                "keyValuePairs[0][year]=2000",
-                "keyValuePairs[1][key]=DateTo",
-                "keyValuePairs[1][day]=1",
-                "keyValuePairs[1][month]=1",
-                "keyValuePairs[1][year]=1999",
+                KeyValuePairs = new List<Dictionary<string, string>>()
+                {
+                    new Dictionary<string, string>()
+                    {
+                        {"key", "DateFrom"}, {"day", "1"}, {"month", "1"}, {"year", "2000"}

```

Comment:
Very minor thing, but I think this would be more readable if each of these pairs were on a new line